docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,159 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Deploying Modules
+
+## 📖 Table of Contents
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
+  * [Creating your Deploy Action](#creating-your-deploy-action)
+  * [Create and Update Your Module Map](#create-and-update-your-module-map)
+    * [Creating a Module Map in Now.sh](#creating-a-module-map-in-nowsh)
+    * [Create an Update Module Script](#create-an-update-module-script)
+    * [Update your Deploy Action to Update your Module Map](#update-your-deploy-action-to-update-your-module-map)
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
+
+# Deploying to Now with GitHub Actions
+
+[Now.sh](https://zeit.co/home) allows you to deploy statics such as your modules. We will accomplish this be using [GitHub Actions](https://github.com/features/actions).
+
+## Creating your Deploy Action
+
+This deploy action uses Zeit's `now-cli` which you will need a [token from Now](https://zeit.co/account/tokens). The below action runs whenever a release is created or a push to master occurs. This will create 
+
+```yml
+name: Deploy
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Cache NPM Dependencies
+      uses: actions/cache@v1
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+    - name: Install Dependencies
+      run: npm install
+      env:
+        NODE_ENV: development
+    - name: Build One App Page Wrapper
+      run: npm run build
+      env:
+        NODE_ENV: production
+    - name: 'Deploy'
+      run: now build --prod --confirm -t $NOW_TOKEN
+      env:
+        NOW_TOKEN: ${{ secrets.NOW_TOKEN }}
+```
+
+If you view the action running, during the `Deploy` step, you should see something like
+
+```bash
+Run now build --prod --confirm -t $NOW_TOKEN
+Now CLI 17.1.1
+The `name` property in now.json is deprecated (https://zeit.ink/5F)
```

Comment:
Can we tell them to do something that _isn't_ deprecated?

Reply:
I can tinker around with that, I got this from someone that works for Zeit. But let me try somethings and update accordingly.

Reply:
@10xLaCroixDrinker so what they are recommending is project linking. What this does is creates a `.now` folder and links it by id instead of by name. It would require the end-user to commit this `.now` folder which when you run `now` locally it automatically adds it to your `.gitignore`. I was talking to Guillermo Rauch who is the CEO of Zeit and he mentioned that they can work on improving that message. So for the time being to run it in a CI/CD pipeline (CircleCI, GitHub Actions, Travis) you have to run it this way.

The issue with project linking is that we need `NODE_ENV=development` for `npm install` but then `NODE_ENV=production` for the bundling of the module. It looks like in their canary version of `now-cli` they allow you to install all the `devDependencies` even when `NODE_ENV=production`. However, not sure if we want to rely on that.